LimeSurvey/LimeSurvey
# MW-514: template files and previews

Integrated the new templates recently received into the project and ensured that their preview is displayed in a proper manner

How to test:

- look into the folders inside assets/templates
- choose one of those, that will be the category
- inside the category folder you will find files, choose one
- encode the string of `category/filename.lss` into a base64 string
- log out with the user
- load the login page, passing a templatetoken parameter to it with the value of the string you just encoded, a sample URL is http://ls-ce/index.php?templatetoken=Y3VzdG9tZXIvTlBTIHN1cnZleSB0ZW1wbGF0ZS5sc3M=&r=admin/authentication/sa/login
- log in with an administrator account
- we expect a popup to appear, confirm the popup
- once the request is completed, we expect the popup to change its size in an animated manner, displaying the preview of the template you just imported
- in the popup, we expect to see the full preview, without welcome view or other unnecessary content
